README.md

```diff
@@ -416,6 +416,18 @@ functions:
     vendor: ./hello-vendor # The option is also available at the function level
 ```
 
+### Copy dependencies instead of linking
+
+Before final packaging, a link is created in .serverless folder for python dependencies.
+If it is not possible to create a symbolic link, dependencies can be copied instead of linked
+whith the foloowing option:
+
+```yaml
+custom:
+  pythonRequirements:
+    useFinalCopy: true
```

Comment:
I'm not sure if this property name is a great choice here - it doesn't really convey the meaning in my opinion - maybe we could use something more descriptive? Like `useSymlinks: false` or something along those lines? What do you think?

Reply:
Property naming is always a difficult task :) Why not `useSymlinks` indeed.